src/pages/home/report/ReportActionsList.tsx

```diff
@@ -223,7 +226,7 @@ function ReportActionsList({
             const isNextMessageRead = !nextMessage || !isMessageUnread(nextMessage, unreadMarkerTime);
             const shouldDisplay = isCurrentMessageUnread && isNextMessageRead && !ReportActionsUtils.shouldHideNewMarker(reportAction);
             const isWithinVisibleThreshold = scrollingVerticalOffset.current < MSG_VISIBLE_THRESHOLD ? reportAction.created < (userActiveSince.current ?? '') : true;
-            return shouldDisplay && isWithinVisibleThreshold;
+            return shouldDisplay && isWithinVisibleThreshold && !lastActionAddedByCurrentUser.current;
```

Comment:
Fixed by changing the check here to return false as long as `lastActionAddedByCurrentUser` exists. What happened before is, we only store the last action in `lastActionAddedByCurrentUser`. So, when we send message 1,2,3, the `lastActionAddedByCurrentUser` will be 3 and `unreadMarkerTime` is updated to 3's `created`.

When we receive the BE response, the 1 `created` value will be bigger than the optimistic 3 `created` value and previously, we only prevent the unread marker to show if `lastActionAddedByCurrentUser` is the same as the action that is being checked.